roles/image-cache/tasks/main.yml

```diff
@@ -10,8 +10,20 @@
     path: "{{ images_dir }}/{{ image_name }}"
   register: image_path
 
+- name: Check if local path exists and copy to the image-cache directory
+  become: true
```

Comment:
Is become really needed?

Reply:
This is because I used `become: true` for the [image-cache directory creation](https://github.com/suse-edge/metal3-demo/blob/main/roles/image-cache/tasks/main.yml#L3), which is because it [by default goes under ](https://github.com/suse-edge/metal3-demo/blob/main/roles/image-cache/defaults/main.yml#L1)`/var/lib/libvirt/images` which is only accessible by root

It would be good to reduce the usage of `become: true` but that would mean opening up permissions on the libvirt directory, or picking a different path (which caused problems with apparmor, but I ended up disabling it anyway since also caused issues with the default pool directory 😬 )

The other issue is rootless podman didn't seem to work out of the box when I tried it on Ubuntu, so we have several inter-related issues to figure out before this can all work cleanly without root permissions.

Reply:
same answer...I will test it again without that part

Reply:
removed from all edge-image-builder part